extensions/MyDashboard/web/js/flags.js

```diff
@@ -16,59 +16,48 @@ $(function () {
         // Common
         var counter = 0;
         var dataSource = {
-            reviews: null,
             requestee: null,
             requester: null
         };
         var dataTable = {
-            reviews: null,
             requestee: null,
             requester: null
         };
-        var hasReviews = !!document.getElementById('reviews_container');
 
-        var updateRequestsTable = function(type) {
+        var updateFlagTable = function(type) {
             if (!type) return;
 
             counter = counter + 1;
 
             var callback = {
                 success: function(e) {
                     if (e.response) {
-                        Y.one('#' + type + '_loading').addClass('bz_default_hidden');
                         Y.one('#' + type + '_count_refresh').removeClass('bz_default_hidden');
                         Y.one("#" + type + "_flags_found").setHTML(
-                            e.response.results.length +
-                            ' request' + (e.response.results.length == 1 ? '' : 's') +
-                            ' found');
+                            e.response.results.length + ' flags found');
```

Comment:
looks like this is a straight revert, which is a problem.

the commit with the phab dashboard also added some quality of life changes to the dashboard as a whole (loading indicators, correct pluralisation, improved error management, etc).  these changes shouldn't be reverted.